src/consul/mod.rs

```diff
@@ -0,0 +1,95 @@
+use std::collections::BTreeMap;
+
+use testcontainers::core::WaitFor;
+use testcontainers::Image;
+
+const DEFAULT_IMAGE_NAME: &str = "hashicorp/consul";
+const DEFAULT_IMAGE_TAG: &str = "1.16.1";
+const CONSUL_LOCAL_CONFIG: &str = "CONSUL_LOCAL_CONFIG";
+
+#[derive(Debug)]
+pub struct Consul {
```

Comment:
Could you add a documentation? We ask this for all new modules

Example https://github.com/testcontainers/testcontainers-rs-modules-community/blob/0963b03da864ae812ce0629917e9de9b69b6c335/src/rabbitmq/mod.rs#L6-L29

Reply:
done